.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.9
+    working_directory: ~/repo
```

Comment:
Do you know why this is necessary ? Have you tried without this ?

Reply:
I have not tried it. It is customarily used.
![image](https://user-images.githubusercontent.com/1130921/39735046-c2ca8718-52b4-11e8-9dd6-7ff12d6646f4.png)
Actually it's not a required param though.

Reply:
Then we can remove the line if it's not useful :)